### ISSUE TITLE
deploy EKS to IAM account

### ISSUE THREAD

**Issue by danielhartnell** [mozilla-itcloud/iam-project#7]
For the time being, I'll plan on using [this Terraform](https://www.terraform.io/docs/providers/aws/guides/eks-getting-started.html) to deploy the cluster.

**Comment by danielhartnell** [comment]
The cluster was deployed successfully with that Terraform. I'll have that up in this repository today:

https://github.com/mozilla-iam/eks-deployment

All up and running. Woohoo! I would like to get Heptio Sonobuoy to successfully return data so we can review the conformance test results and compare the results with our existing hardened cluster.

```
➜  eks-deployment git:(master) ✗ kubectl version --short
Client Version: v1.10.3
Server Version: v1.10.3
➜  eks-deployment git:(master) ✗ kubectl cluster-info
Kubernetes master is running at https://****.***.us-west-2.eks.amazonaws.com

To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
➜  eks-deployment git:(master) ✗
```

**Comment by danielhartnell** [comment]
The cluster can be consistently deployed now. I have modified that Terraform to support multiple clusters by providing different cluster names. Prior to that, it would fail because of hard coded resources names in AWS.

I have had trouble running Sonobuoy, possibly because the masters are managed by AWS, but I was able to run `kube-bench` against the worker nodes (also failed against the masters) and share the results with the team.
